File: Data Structures & Algorithms/is-anagram/submission-1.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if(s.length()!=t.length())
        return 0;
        map<char ,int> m1,m2;
        for(int i;i<s.length();i++)
        {
            m1[s[i]]++;
            m2[t[i]]++;
        }
        if(m1==m2)
        return 1;
        return 0;
        
    }
};

File: Data Structures & Algorithms/flood-fill/submission-1.py
class Solution:
    def floodFill(self, image: List[List[int]], sr: int, sc: int, color: int) -> List[List[int]]:
        ogcolor = image[sr][sc]
        visited = set()
        
        def ff(image,sr,sc, color):
            ROW,COL = len(image) , len(image[0])
            if sr >= ROW or sc >=COL or min(sr,sc)<0 or (sr,sc) in visited or image[sr][sc]!=ogcolor:
                return
            
            image[sr][sc]=color
            visited.add((sr,sc))
            ff(image,sr+1,sc,color)
            ff(image,sr,sc+1,color)
            ff(image,sr,sc-1,color)
            ff(image,sr-1,sc,color)
            visited.remove((sr,sc))
            
        ff(image,sr,sc,color)
        
        return image
            